tests/helpers.bash

```diff
@@ -5,6 +5,7 @@ IMGTYPE_BINARY=${IMGTYPE_BINARY:-$(dirname ${BASH_SOURCE})/../imgtype}
 TESTSDIR=${TESTSDIR:-$(dirname ${BASH_SOURCE})}
 STORAGE_DRIVER=${STORAGE_DRIVER:-vfs}
 PATH=$(dirname ${BASH_SOURCE})/..:${PATH}
+OCI=$(${BUILDAH_BINARY} info --format '{{.host.OCIRuntime}}' || command -v runc || command -v crun)
```

Comment:
What benefit do the two `command -v`s add? Shouldn't we trust what buildah itself reports?

Reply:
There were tests that would pass if no OCI Runtime is defined.  I was not sure what happens in that case, if you don't have crun or runc on a system.  Then test would be cancelled.  This way we are more likely to get something if buildah info fails to find one.